src/TestHarness/Main.Designer.cs

```diff
@@ -102,9 +102,9 @@ private void InitializeComponent()
             // 
             // Output
             // 
-            this.Output.Anchor = ((System.Windows.Forms.AnchorStyles)((((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Bottom)
-                        | System.Windows.Forms.AnchorStyles.Left)
-                        | System.Windows.Forms.AnchorStyles.Right)));
+            this.Output.Anchor = ((System.Windows.Forms.AnchorStyles)((((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Bottom) 
+            | System.Windows.Forms.AnchorStyles.Left) 
+            | System.Windows.Forms.AnchorStyles.Right)));

```

Comment:
Ignore me if this is auto-generated, but the parentheses separating the different flags seems really weird to me, as well as continuing lines without any indentation.

Reply:
This is autogenerated. I didn't even look at this file to be honest :D